core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -115,6 +115,7 @@ private[spark] class Executor(
   if (!isLocal) {
     env.metricsSystem.registerSource(executorSource)
     env.blockManager.initialize(conf.getAppId)
+    env.metricsSystem.registerSource(env.blockManager.shuffleMetricsSource)
```

Comment:
super nit: can you put the `reigsterSource()` calls next to each other?

Reply:
Sure, let me update it.